ReSharper.FSharp/src/FSharp.Psi.Features/src/Daemon/Analyzers/LambdaAnalyzer.fs

```diff
@@ -96,11 +96,22 @@ type LambdaAnalyzer() =
         | :? FSharpMemberOrFunctionOrValue as m ->
             m.IsMember &&
             let lambdaPos = if isNotNull appTuple then appTuple.Expressions.IndexOf(argExpr) else 0
-            let args = m.CurriedParameterGroups
-            if args.[0].Count <= lambdaPos then false else
-            let argDecl = args.[0].[lambdaPos]
+            let args = m.CurriedParameterGroups.[0]
+            if args.Count <= lambdaPos then false else
+            let argDecl = args.[lambdaPos]
             let argDeclType = argDecl.Type
-            argDeclType.HasTypeDefinition && (getAbbreviatedEntity argDeclType.TypeDefinition).IsDelegate
+            let argIsDelegate =
+                argDeclType.HasTypeDefinition && (getAbbreviatedEntity argDeclType.TypeDefinition).IsDelegate
+
+            if argIsDelegate then
+                let apparentEntity = m.ApparentEnclosingEntity
+                let hasOverload =
```

Comment:
What particular overload this name is about?

Reply:
any overload corresponding to the criterion